test/benchmarks/material/button-harness/testbed-benchmark-utilities.ts

```diff
@@ -0,0 +1,32 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+/**
+ * Records the performance of the given function.
+ *
+ * @param id A unique identifier.
+ * @param callback A function whose performance will be recorded.
+ */
+export async function benchmark(id: string, callback: Function) {
+  const t0 = performance.now();
```

Comment:
I think `console.time` should work in browsers too?

Reply:
Coming back after looking at Karma. See my reply on the `BUILD.bazel` file. We should add a comment here.